README.md

```diff
@@ -1,5 +1,12 @@
 # ROS_PRBT
 
+
+### Build Status
+
+| Kinetic |
+| ------ |
+| [![Build Status](https://travis-ci.org/PilzDE/pilz_robots.svg?branch=kinetic-devel)](https://travis-ci.org/PilzDE/pilz_robots) |
```

Comment:
I'd prefer the status at the bottom of the page. At least put the build status below the outermost headline (`## Package`). Actually

Reply:
Done.